draft-gpew-priv-ppm.md

```diff
@@ -1075,6 +1074,8 @@ enum {
 } AggregateReqType;
 
 struct {
+    TaskID task_id;
+    AggregationJobID job_id;
```

Comment:
nit: we can remove the note below this struct now

Reply:
Done